cmd/copy.go

```diff
@@ -1652,6 +1652,9 @@ func (cca *CookedCopyCmdArgs) processCopyJobPartOrders() (err error) {
 	if err != nil {
 		if err == NothingToRemoveError || err == NothingScheduledError {
 			return err // don't wrap it with anything that uses the word "error"
+		} else if cca.S2sPreserveBlobTags && strings.Contains(err.Error(), "AuthorizationPermissionMismatch") {
+			// If S2sPreserveBlobTags is true, return a more descriptive error message indicating permission issue.
+			return fmt.Errorf("Failed to set blob tags due to permission error: %s.\n", err)
```

Comment:
Is this the only way we can get this back? Could this not occur later in a job, after enumeration?

Reply:
this is the only place we can  get this back as of now , even after job enumeration we are coming here.